examples/website/point-cloud-laz/app.js

```diff
@@ -157,7 +158,7 @@ class Example extends PureComponent {
       <DeckGL
         width={width}
         height={height}
-        views={view}
+        views={[view]}
```

Comment:
`views` accepts both View object and array.

Reply:
if that's the case, there is a bug in our core module which doesn't accept a view object.